packages/adapter-static/index.js

```diff
@@ -26,13 +27,13 @@ export default function ({ pages = 'build', assets = pages, fallback, precompres
 
 			if (precompress) {
 				if (pages === assets) {
-					builder.log.minor('Compressing assets and pages');
+					builder.log(`Compressing assets and pages to directory: ${assets}`);
```

Comment:
Is there a reason to remove `.minor`?

Reply:
No — these should only be visible with `--verbose`. Rather than changing all these I think it would be better to have a single `builder.log` at the end of the process like

> Wrote site to "${pages}"

or

> Wrote pages to "${pages}" and assets to "${assets}"

Reply:
Mostly because how verbose vite's output is when building the site (listing every file built).